toolkits/landcover/test/int/composites.int.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var lct = require('../../api.js');
+var TestImage = require('../helpers/test-image.js');
+
+withEarthEngine('Composites', function() {
+  it('createTemporalComposites()', function(done) {
+    var testCollection = ee.ImageCollection([
+      TestImage.create({value: 100}, '2015-12-01'),
+      TestImage.create({value: 150}, '2015-12-15'),
+      TestImage.create({value: 200}, '2016-01-01'),
+      TestImage.create({value: 250}, '2016-01-15'),
+      TestImage.create({value: 300}, '2016-02-01'),
+      TestImage.create({value: 350}, '2016-02-15'),
+      TestImage.create({value: 400}, '2016-03-01'),
+      TestImage.create({value: 450}, '2016-03-15'),
+      TestImage.create({value: 500}, '2016-04-01'),
+      TestImage.create({value: 550}, '2016-04-15')
+    ]);
+
+    var result = lct.Composites.createTemporalComposites(
+        testCollection,
+        /* startDate= */ '2016-01-01',
+        /* count= */ 3,
+        /* interval= */ 1,
+        /* intervalUnits= */ 'month', ee.Reducer.mean());
+
+    TestImage.reduceConstants(result.select('value'))
+        .evaluate(function(actual, error) {
+          expect(error).toBeUndefined();
+          // Mean value in each valid 3 month range.
+          expect(actual).toEqual([{value: 225}, {value: 325}, {value: 425}]);
+          done();
+        });
+  });
+
+  it('createMedioidComposite()', function(done) {
+    // Median idx = 3, so we expect the pixel(s) with idx closest to that
```

Comment:
The point of medioid is to take the closest one.  When you have an odd number and one of them is an exact match, you can't really tell if the medioid part is working vs median or mean.  Recommend using an even number by removing the matching "middle" one and staggering the other values a little to ensure which of the two around the middle is closest.

Reply:
Gotcha. New PR coming your way.